docs/basics/annotation.md

```diff
@@ -15,16 +13,34 @@ _Tip: To begin annotating an image, double click on the desired image in the cur
 
 ![Controls for the Management of Annotations](/img/annotate/annotate_management.png)
 
+![Annotate Add Button](/img/annotate/annotate_add.png)
```

Comment:
all these images appear broken - not sure if they are or just in my preview

Reply:
They are working in the built preview: https://gray-island-021371003-38.westeurope.1.azurestaticapps.net/docs/basics/annotation